docs/devel/testing.md

```diff
@@ -165,31 +198,51 @@ is [table driven testing](https://github.com/golang/go/wiki/TableDrivenTests)
 
 ### Install etcd dependency
 
-Kubernetes integration tests require your PATH to include an [etcd](https://github.com/coreos/etcd/releases) installation.
-Kubernetes includes a script to help install etcd on your machine.
+Kubernetes integration tests require your PATH to include an
+[etcd](https://github.com/coreos/etcd/releases) installation. Kubernetes
+includes a script to help install etcd on your machine.
 
 ```sh
 # Install etcd and add to PATH
 
 # Option a) install inside kubernetes root
-cd kubernetes
 hack/install-etcd.sh  # Installs in ./third_party/etcd
```

Comment:
Maybe add a line or two about setting ``ETCD_VERSION`` env? The current version is fixed to ``2.2.1``. Would be worth to mention one can check ``https://github.com/coreos/etcd/releases`` and set the env to newer version, e.g. ``2.3.5``.